db-engine/src/main/java/com/splicemachine/db/iapi/store/access/TransactionController.java

```diff
@@ -247,66 +247,95 @@ gaining access to resources (conglomerates, scans, etc.) in the transaction
      * wait if the application has not set a 0 timeout and if the call does
      * not have a wait parameter (like OpenConglomerate.fetch().
      **/
-	int OPENMODE_LOCK_NOWAIT           = 0x00000080;
+    int OPENMODE_LOCK_NOWAIT           = 0x00000080;
 
     /**
      * Constants used for the countOpen() call.
      **/
-	int OPEN_CONGLOMERATE   = 0x01;
+    int OPEN_CONGLOMERATE   = 0x01;
     int OPEN_SCAN           = 0x02;
     int OPEN_CREATED_SORTS  = 0x03;
     int OPEN_SORT           = 0x04;
     int OPEN_TOTAL          = 0x05;
 
 
-	byte IS_DEFAULT	=	(byte) 0x00; // initialize the flag
-	byte IS_TEMPORARY	=	(byte) 0x01; // conglom is temporary
-	byte IS_KEPT		=	(byte) 0x02; // no auto remove
+    byte IS_DEFAULT	=	(byte) 0x00; // initialize the flag
+    byte IS_TEMPORARY	=	(byte) 0x01; // conglom is temporary
+    byte IS_KEPT		=	(byte) 0x02; // no auto remove
 
+    /**
+     * Conflict resolution strategies.
+     */
+    enum ConflictResolutionStrategy {
```

Comment:
we also have a ConflictResolutionStrategy.java, do we need both?

Reply:
Thanks, this is a left over from a previous implementation approach that is refactored. I just removed it.